_posts/2020-09-07-introduction-to-system-tests.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: post
+title:  "Introduction to system tests"
+date: 2020-09-07
+author: maroš_orsák
+---
+
+Lessons learn from past years shown that testing is the essential part of the software cycle. We have known many 
```

Comment:
sorry to be pedantic on this, one sentence per line, please :-)

Reply:
No problem :) I will update it